conf/parse.go

```diff
@@ -127,6 +129,10 @@ func cleanupUsedEnvVars(m map[string]any) {
 	}
 }
 
+func DisableConfigurationDigest(disable bool) {
```

Comment:
Since this would become a part of the exported API, I think it would be best if we add comments on why you might want to (or moreover, why you probably shouldn't) use it. It's also not thread-safe if someone does decide to fiddle with things with a running embedded server.

Reply:
You know, I don't like it either, and I had started with a private variable in the server that was set it sets, but I would have had the same problem with the tests in the `test` package.

What I am going to do, is likely revert the changes unrelated to the JS command line parameters and on a different branch, run all tests with a debug statement in place where we ignore and have a sense of how many tests need review and go from there.

I am going to switch this PR to Draft until I have a chance to update it later today.